.travis.yml

```diff
@@ -1,18 +1,46 @@
 language: python
 
-python:
-- '3.6'
+env:
+  - PYTHON=python
+
+matrix:
+  include:
+    - os: linux
+      python: 3.6
+    - os: linux
+      # Get Python 3.7 from Xenial
+      # https://github.com/travis-ci/travis-ci/issues/9815
+      python: 3.7
+      dist: xenial
+    - os: windows
+      # Get Python 3.7 from Chocolatey
+      language: sh
+      python: 3.7
+      env: PYTHON=py
```

Comment:
I suppose using just `python` on Windows didn't work?

Reply:
It works! Thanks :)
As you can imagine, getting this to work was a lot of trial and error. After that I tend to be blind to the details :)